pkg/cortex/cortex.go

```diff
@@ -199,10 +205,13 @@ func New(cfg Config) (*Cortex, error) {
 
 	cortex.setupAuthMiddleware(&cfg)
 
-	if err := cortex.init(&cfg, cfg.Target); err != nil {
+	serviceMap, err := cortex.initModuleServices(&cfg, cfg.Target)
+	if err != nil {
 		return nil, err
 	}
 
+	cortex.serviceMap = serviceMap
+	cortex.server.HTTP.Handle("/services", http.HandlerFunc(cortex.servicesHandler))
```

Comment:
Would be nice to document the endpoint in `docs/apis.md`

Reply:
I think we should beautify it little bit first, and perhaps also include subservices. Right now it's useful as a debugging tool for us.

Reply:
Added to TODOs list as followup task.